WTWSMS/common/scripted_triggers/WtWSMS_scripted_triggers.txt

```diff
@@ -28,6 +28,7 @@ has_romance_culture_trigger = {
 		culture = helleno_thracian
 		culture = massalian
 		culture = romano_dacian
+		culture = spartan
```

Comment:
If they are revived in this time period, they should be able to reform their empire of old, and in this case, usurp the Roman Empire. All Greeks have heritage from Rome, including the Tsakonians who can become Spartan.